### PR TITLE
Quick Actions: Open style tab after adding a text element

### LINKED ISSUE #11291: Text tab is not displayed when adding a text using quick add while being on another tab
## Bug Description

If the user is on the Media tab and uses the quick add button to add a text, the Text tab is not displayed. Please see video below.

## Expected Behaviour

When the user adds a text using the T+ button on the quick action menu, the Text tab should be displayed.

## Steps to Reproduce

1. Go to the stories editor
2. Click on the stories editor
3. Click on the T+ button on the quick add menu - Text tab is displayed
4. Click on any other tab
5. Click on the stories editor
6. Click on the T+ button on the quick add menu - Text tab is NOT displayed

## Screenshots

https://images.zenhubusercontent.com/235435637/03b97eac-2932-4f17-9426-90b06442094a/text_t_.mp4

## Additional Context

- Plugin Version:
- WordPress Version:
- Operating System:
- Browser:

### PR DESCRIPTION
**Pull request by miina**
## Relevant Technical Choices
Previously, the highlight was on the text sets panel which doesn't really make sense since it's an insertion panel for text sets and not related to styling the newly added text element.

## Testing Instructions

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1. Unselect all elements
2. Click the quick-action to insert text.
3. Verify the Style tab gets opened
4. Try again the same to verify it works the second time, too

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [ ] I have tested this code to the best of my abilities
- [ ] I have verified accessibility to the best of my abilities ([docs](https://github.com/googleforcreators/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [ ] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [ ] This code is covered by automated tests (unit, integration, and/or e2e) to verify it works as intended ([docs](https://github.com/googleforcreators/web-stories-wp/tree/main/docs#testing))
- [ ] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #11291

### CODE REVIEW

File: packages/story-editor/src/app/highlights/quickActions/useQuickActions.js [hunk 1]
```diff
@@ -484,15 +484,14 @@ const useQuickActions = () => {
...
-    handleFocusTextSetsPanel,
     handleFocusCaptionsPanel,
   } = useMemo(
     () => ({
       handleFocusAnimationPanel: handleFocusPanel(states.ANIMATION),
       handleFocusLinkPanel: handleFocusPanel(states.LINK),
       handleFocusPageBackground: handleFocusPanel(states.PAGE_BACKGROUND),
-      handleFocusTextSetsPanel: handleFocusPanel(states.TEXT_SET),
       handleFocusCaptionsPanel: handleFocusPanel(states.CAPTIONS),
+      handleFocusStylePanel: handleFocusPanel(states.STYLE_PANE),
```
Comment (barklund) [inline packages/story-editor/src/app/highlights/quickActions/useQuickActions.js]: Is this doing anything?

**Review by barklund — APPROVED** [review]
✅

**Comment by kkalarickal** [comment]
Confirmed that using the floating/quick-action menu to click the T+ icon 
- adds a "Fill in some text" placeholder on the page
- switches context on the left hand sidebar to be under Style
- it is possible to edit the style for the text element
- clicking outside the element and repeating the T+ click brings back the Style menu
[Uploading... image.png]

**Review by swissspidy — APPROVED** [review]

**Comment by swissspidy** [comment]
LGTM, just needs tests and test fixes.

**Review by spacedmonkey — APPROVED** [review]

**Pull request merged**